datasets/bgt/bgt_roads_demo.csv

```diff
@@ -1,26 +1,29 @@
+bgt_type,polygon,x_min,y_max,x_max,y_min

```

Comment:
Het lijkt erop dat er meer veranderd is aan dit bestand dan alleen de column name "bgt_name"